doc/integrator/extend_application.rst

```diff
@@ -40,17 +40,243 @@ In the ``vars.yaml`` file your interface should be declared like that:
    interfaces:
      - name: desktop
        type: canvas
+       layout: desktop
        default: true
 
+The ``name`` is the interface name as usual.
+The ``type`` should be set to 'canvas' to be able to get the canvas based interface present in the config image.
+The ``layout`` is used to the the JavaScript and CSS files from ngeo.
```

Comment:
I suppose this is only relevant for canvas, and possible values are [desktop|mobile|iframe]
```suggestion
The ``layout`` is used to get the JavaScript and CSS files from ngeo.
```